.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: php
+
 php:
+  - 5.6
   - 5.5
   - 5.4
   - hhvm
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - php: hhvm
```

Comment:
I'd rather not just shut it off, I'm working on getting the Travis config updated and hoping that in doing so we'll be able to successfully build on hhvm again. This is likely to conflict with this change, in the future it'd be good to isolate changes to this kind of file from actual source code changes to avoid conflicts.